src/common.hh

```diff
@@ -60,24 +60,22 @@ struct ThreadPool final
     ThreadPool(size_t n_threads, std::function<void(const std::string&)> err);
     ~ThreadPool() noexcept;
 
-    void start();
     void push_to_job_queue(JobT&& job);
     void await_stop() noexcept;
```

Comment:
Optional

Add a comment here to explain that calling this will wait/block until pending jobs are complete. Also, after this is complete, no new jobs can be submitted.